### DISCUSSION TITLE
How to use material 3 design?

### DISCUSSION THREAD

**Discussion by icewyvrn** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3610] [Q&A] [answered]
I am using version `5.0.0`
I already have this in my `App.xaml`
`<ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesign3.Defaults.xaml" /> `

But it's still showing old design
![image](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/assets/41411360/c36c49ee-8442-467e-8469-680c0c1d806e)

**Comment by MichelMichels** [comment, marked as answer]
The Material Design 3 control library is in development and far from complete. If the control you're looking for is not in the demo, it probably does not exist yet and falls back to the Material Design 2 control.
